src/test/java/org/jenkinsci/plugins/ewm/steps/ExwsAllocateStepTest.java

```diff
@@ -159,11 +159,13 @@ public void upstreamJobHasNoStableBuild() throws Exception {
     @Test
     public void upstreamJobHasNoActionRegistered() throws Exception {
         String jobName = randomAlphanumeric(10);
-        createWorkflowJobAndRun(jobName, "echo 'hello world'");
+        upstreamRun = createWorkflowJobAndRun(jobName, "echo 'hello world'");
         createDownstreamJobAndRun(jobName);
 
         j.assertBuildStatus(FAILURE, downstreamRun);
-        j.assertLogContains(format("The Jenkins job '%s' does not have registered any 'External Workspace Allocate' action. Did you run exwsAllocate step in the upstream job?", jobName), downstreamRun);
+        j.assertLogContains(format("This downstream job has to allocate the same external workspace used by the upstream job. " +
+                "To do so, the exwsAllocate step must be called in the upstream job. " +
+                "Please call the exwsAllocate step in the upstream job, because the build '%s' doesn't have such calls.", upstreamRun), downstreamRun);
```

Comment:
I commented on the actual change in PR #8 . I find it difficult to come up with a good error message for this error condition, but please see my comment, I think I have nailed it.